.github/workflows/newsletter-issue.yml

```diff
@@ -0,0 +1,33 @@
+name: Newsletter Issue
+
+on:
+  schedule:
+  - cron: "30 0 1 * *"    # At 00:30 on day-of-month 1.
+
+jobs:
+  issue:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Event edition
+        id: event
+        run: echo "::set-output name=edition::$(date +'%B-%Y')"
```

Comment:
```suggestion
      - name: Newsletter edition
        id: edition
        run: echo "::set-output name=date::$(date +'%B %Y')"
```
How about this?

Reply:
Its more general term we are started using like even and edition (Personally don't want to separate Newsletter). I would like go with `event` and its `edition`.
like PythonPune July 2020 edition.
PythonPune meet-up is our `event` and `July 2020` is our edition.

Yup I will remove `-` as we are not using it for consistency.

Reply:
Okay, fair enough.